hw1/yannik-st/countline.py

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+"true" '''\'
+if [ "$PYTHON_BIN" = "python2" -o "$PYTHON_BIN" = "python3" ];
```

Comment:
actually, you just need to let the system to run the `$PYTHON_BIN`. maybe there is python4 defined? who knows.